2003/mtjin/AndroidArchitectureStudy/app/src/main/java/com/mtjin/androidarchitecturestudy/utils/PreferenceManager.kt

```diff
@@ -0,0 +1,25 @@
+package com.mtjin.androidarchitecturestudy.utils
+
+import android.content.Context
+import android.content.SharedPreferences
+
+object PreferenceManager {
+    private const val MOVIE_SEARCH_APP = "MOVIE_SEARCH_APP"
+    const val AUTO_LOGIN_KEY = "AUTO_LOGIN_KEY"
+
+    private fun getPreferences(context: Context): SharedPreferences? {
```

Comment:
반환값이 Nullable일 필요가 있을까요?
```suggestion
    private fun getPreferences(context: Context): SharedPreferences {
```

Reply:
수정했습니다.

d13bbbf